proto/mls/message_contents/content.proto

```diff
@@ -39,4 +39,20 @@ message EncodedContent {
   optional Compression compression = 5;
   // encoded content itself
   bytes content = 4;
-}
\ No newline at end of file
+}
+
+// An MlsEnvelope is the outermost payload that gets encrypted by MLS
+message MlsEnvelope {
+  // Version 1 of the encrypted envelope
+  message V1 {
+    // Expected to be EncodedContent
+    bytes content = 1;
```

Comment:
Any reason we have to serialize the EncodedContent to bytes here, rather than just nesting it as-is?

Reply:
I was going to post that as a question to the group.

Right now, platform SDKs handle all decoding of content. They expect messages as bytes and decode to `EncodedContent` and then decode the inner content using content types.

While we could nest `EncodedContent` here, we would need to re-serialize the `EncodedContent` when we store it in the DB. And to send a message we'd need to deserialize the `EncodedContent` passed to `libxmtp` by the caller to create this payload.

Reply:
I see! Good catch - fine with leaving it as bytes.